### ISSUE TITLE
Does not work with *.html on Windows

### ISSUE THREAD

**Issue by ashjs** [donpark/html2jade#90]
Hello,

How I use : html2jade --tabs --donotencode docs/*.html

I have this error : input file doesn't exist: docs/*.html

Kindly.

**Comment by barecool** [comment]
works on windows if using msygit and running htm2jade on gitbash for windows. but not on the regular command prompt

**Comment by donpark** [comment]
@barecool, that's true for me too. if node.js apps can run on regular command prompts, it's news to me.
